UDPServer/SiceUDPServer.cs

```diff
@@ -85,5 +82,19 @@ public void Dispose()
         {
             client.Dispose();
         }
+
+        public void Handle(ServerCommandEvent command)
+        {
+            if(command.Command)
+            {
+                ConnectionIP = command.ConnectionIP;
+                ConnectionPort = command.ConnectionPort;
+                this.Connect();
+            }
+            else
+            {
+                this.Disconnect();
+            }
```

Comment:
Nothing wrong with this code, but you could write it guard condition style.
```
if(!command.Command){
this.Disconnect()
return;
}

ConnectionIP = command.ConnectionIP;
ConnectionPort = command.ConnectionPort;
this.Connect();
```

Reply:
Noted